pdf.go

```diff
@@ -870,6 +909,10 @@ func (w *pdfPageWriter) WriteText(TJ ...interface{}) {
 	units := float64(w.font.sfnt.UnitsPerEm())
 
 	first := true
+	escapedWriter := stringBytesEscaper{
+		w:           w,
+		escapeChars: []byte{'\\'},
```

Comment:
Wouldn't it be more useful to specify which range of characters need to be escaped instead of which escape character ti use? Either define both (very general), or neither (specialized to this case).

Reply:
You are right, le me change this.